src/Symfony/Component/Cache/Adapter/ApcuAdapter.php

```diff
@@ -13,16 +13,22 @@
 
 use Symfony\Component\Cache\CacheItem;
 use Symfony\Component\Cache\Exception\CacheException;
+use Symfony\Component\Cache\Marshaller\MarshallerInterface;
 
 /**
  * @author Nicolas Grekas <p@tchwork.com>
  */
 class ApcuAdapter extends AbstractAdapter
 {
+    /**
+     * @var MarshallerInterface|null
+     */
```

Comment:
```suggestion
```

We don't add phpdoc on properties, it's inferred from the constructor by static analysers and most IDEs.